nautobot/dcim/models/device_components.py

```diff
@@ -495,6 +499,22 @@ class BaseInterface(RelationshipModel):
         verbose_name="MTU",
     )
     mode = models.CharField(max_length=50, choices=InterfaceModeChoices, blank=True)
+    parent_interface = models.ForeignKey(
+        to="self",
+        on_delete=models.SET_NULL,
+        related_name="child_interfaces",
+        null=True,
+        blank=True,
+        verbose_name="Parent interface",
+    )
+    bridge = models.ForeignKey(
+        to="self",
+        on_delete=models.SET_NULL,
+        related_name="bridge_interfaces",
```

Comment:
As in my previous comment, I *think* `bridged_interfaces` would be more accurate here, but would appreciate feedback.